pom.xml

```diff
@@ -74,6 +79,18 @@
                 <version>1.1.0</version>
                 <scope>provided</scope>
             </dependency>
+
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>${slf4j.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>ch.qos.logback</groupId>
+                <artifactId>logback-classic</artifactId>
+                <version>${logback.version}</version>
+            </dependency>
```

Comment:
I'm wondering if we want to declare the test scope here already, to make sure it's not accidentally exposed...

Reply:
I mean the reasoning was that this is a library so it should not expose any specific implementation, but I agree that this might be limiting :)